python/gps/algorithm/policy/tf_policy.py

```diff
@@ -28,6 +28,7 @@ def __init__(self, dU, obs_tensor, act_op, var, sess, device_string):
         self.chol_pol_covar = np.diag(np.sqrt(var))
         self.scale = None  # must be set from elsewhere based on observations
         self.bias = None
+        self.st_idx = None
```

Comment:
What is st_idx? Is it the idx corresponding to the state? If so, maybe name it x_idx or state_idx instead?